docs/workforce-management-framework/topics/pro-keycloak-implementation.adoc

```diff
@@ -5,44 +5,56 @@
 == Prerequisites
 
 . Ensure you have completed xref:getting-started[Getting Started]
-. Docker (tested with version: {WFM-RC-DockerVersion})
+. A {Keycloak} server
 
-== Using {Keycloak} to secure endpoints
+== Using {Keycloak} to secure Express endpoints
+NOTE: The {PROJECT_NAME} applications uses the {Keycloak} Node.js and Javascript Client Adapters. For more information,
+see xref:{context}-understanding-keycloak-client-adapters[Understanding {Keycloak} Client Adapters].
 
-[id='{context}-initializing-keycloak']
+[id='{context}-implement-endpointsecurity-interface']
 [discrete]
-== Initializing {Keycloak}
+== Implement the EndpointSecurity Interface
 
-See the link:{WFM-RC-CoreURL}{WFM-RC-Branch}/demo/server/src/modules/keycloak/index.ts[Keycloak code] for reference.
-
-[id='{context}-importing-the-endpointsecurity-interface']
-[discrete]
-== Importing The EndpointSecurity Interface
-
-Import the link:{WFM-RC-CoreURL}{WFM-RC-Branch}/cloud/auth/src/EndpointSecurity.ts[EndpointSecurity Interface] and implement it.
+. Import the link:{WFM-RC-CoreURL}{WFM-RC-Branch}/cloud/auth/src/EndpointSecurity.ts[EndpointSecurity Interface].
++
+[source,typescript]
+----
+import { EndpointSecurity } from '@raincatcher/express-auth';
+----
++
+. Implement the EndpointSecurity interface
++
+See the sample link:{WFM-RC-CoreURL}{WFM-RC-Branch}/demo/server/src/modules/keycloak/index.ts[Keycloak implementation] of
+the EndpointSecurity interface as a reference on how this can be implemented.
 
-[id='{context}-importing-keycloak-init']
+[id='{context}-enable-keycloak-security-middleware']
 [discrete]
-== Importing {Keycloak} _init_
+== Enable {Keycloak} Security Middleware for Route Protection
+. Instantiate {Keycloak} and install the {Keycloak} middleware to your express app.
++
+This is implemented in the link:{WFM-RC-CoreURL}{WFM-RC-Branch}/demo/server/src/modules/keycloak/index.ts[Keycloak implementation]
```

Comment:
Invalid link - we should not point to core repo!
💯  Really important to maintain our release integrity. 

ping @paolobueno @austincunningham @witmicko